docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -328,6 +348,11 @@
       padding: 0;
       top: 0;
       left: 0;
+
+      .hb-dark-pattern & .field-content,
+      .hb-dark-inversion .views-element-container & .field-content {
+        color: inherit;
```

Comment:
This makes the month and the date inside the tile white. Perhaps remove lines 352 thru 355 and apply `color: inherit` to only `.hb-card__date-tile-day` on line 384.

Reply:
I ended up applying this just to date as suggested but used primary as it was getting that initially and I felt like being heavy-handed was needed. Hb-raised-cards have its own styling of this because of how the classes are working with each card setup and dark inversion hierarchy.